SPECS/kernel/kernel.signatures.json

```diff
@@ -1,9 +1,9 @@
 {
- "Signatures": {
-  "cbl-mariner-ca-20211013.pem": "5ef124b0924cb1047c111a0ecff1ae11e6ad7cac8d1d9b40f98f99334121f0b0",
-  "config": "ee89feb370e6bca43095336401febf7278fc33e82cc61771a7bc3d56843f6c02",
-  "config_aarch64": "be8c718c2888d3d2eaa5f384589a5ac529e87aa2de5a95643f58caf234e5802a",
-  "kernel-5.15.48.1.tar.gz": "de79bc5d2bc0943e3efa4085f6066192d6df0478235999de204f6a60b46e2f90",
-  "sha512hmac-openssl.sh": "02ab91329c4be09ee66d759e4d23ac875037c3b56e5a598e32fd1206da06a27f"
- }
-}
+  "Signatures": {
+    "cbl-mariner-ca-20211013.pem": "5ef124b0924cb1047c111a0ecff1ae11e6ad7cac8d1d9b40f98f99334121f0b0",
+    "config": "0678916a63d9249eac29ce5a615bfb80b31fecb9dc1c27a1c7f860ab969cb68a",
+    "config_aarch64": "a9fac2c6711d1364749d49131b25e58ddcf3134d07f59861f1393a51aaa3ed04",
+    "sha512hmac-openssl.sh": "02ab91329c4be09ee66d759e4d23ac875037c3b56e5a598e32fd1206da06a27f",
```

Comment:
Just curious why we have diffs in spaces and ordering.

Reply:
because the majority of this update was generated by Nicolas' kernel update pipeline :)